appknox/appknox.go

```diff
@@ -244,6 +244,12 @@ func (c *Client) Do(ctx context.Context, req *http.Request, v interface{}) (*Res
 	return response, err
 }
 
+// DoTxt sends an API request and returns the API response.
+// The API response is plaintext.
+func (c *Client) DoTxt(request *http.Request) (*http.Response, error) {
```

Comment:
This method is doing nothing other than calling `client.Do` method. Why do we need to have this separate method?

Reply:
Removed this method